apps/bridge-dapp/webpack.dev.js

```diff
@@ -17,7 +17,7 @@ module.exports = (env, config) => {
     devServer: {
       hot: true,
       open: false,
-      port: 3000,
+      port: process.env.PORT_BRIDGE_DAPP,
```

Comment:
Consider adding a fallback port here if the env is not provided.

Reply:
i've added fallback [here](https://github.com/webb-tools/webb-dapp/pull/848/commits/c1abc8a3b4574e07cf06d2c7e34b91370b27bd16#diff-daf66e68960d1fd2026bc36b9617598b4f3d60e0f5bc7107ce3ecf66f7bdd0ecR20)